vulnerabilities/tests/pipelines/test_remove_duplicate_advisories.py

```diff
@@ -125,4 +130,4 @@ def test_recompute_content_ids(self):
         # Check that content ID was updated
         advisory.refresh_from_db()
         expected_content_id = compute_content_id(advisory_data=self.advisory_data)
-        self.assertNotEqual(advisory.unique_content_id, expected_content_id)
+        self.assertEqual(advisory.unique_content_id, expected_content_id)
```

Comment:
Why this change of assertion ?

Reply:
To make sure RemoveDuplicateAdvisoriesPipeline() has repopulated the content id using vulnerabilities.utils.compute_content_id